packages/tradershub/src/features/cfd/modals/MT5PasswordModal/ButtonGroups/SuccessButtonGroup.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { useActiveTradingAccount } from '@deriv/api';
+import { Provider } from '@deriv/library';
+import { Button } from '@deriv/quill-design';
+import { ButtonGroup } from '../../../../../components';
+
+const SuccessButtonGroup = () => {
+    const { hide } = Provider.useModal();
+    const { data: activeTrading } = useActiveTradingAccount();
+    const isDemo = activeTrading?.is_virtual;
+    const history = useHistory();
+
+    if (isDemo) {
+        return (
+            <Button onClick={hide} size='lg'>
+                OK
+            </Button>
+        );
+    }
+    return (
+        <ButtonGroup className='justify-center w-full'>
+            <Button className='rounded-200' colorStyle='black' onClick={() => hide()} size='lg' variant='secondary'>
+                Maybe later
+            </Button>
+            <Button
+                className='rounded-200'
+                onClick={() => {
+                    hide();
+                    history.push('/cashier/transfer');
```

Comment:
The actual behaviour for this is to open transfer popup, right?
if yes please remove this history push and add a comment for opening transfer modal

Reply:
the transfer modal is opened if "transfer" is clicked on tradershub but in this case it is through success modal after account creation and according to production as well it is redirected to the transfer page @mahdiyeh-deriv

Reply:
this opens transfer modal
<img width="50%" alt="image" src="https://github.com/binary-com/deriv-app/assets/108507236/9204575f-6205-4b23-85d8-b8819afa4dc5">

this redirects to transfer page
<img width="50%" alt="image" src="https://github.com/binary-com/deriv-app/assets/108507236/1010dc9e-e512-4787-9c71-e4b371619a03">